src/main/kotlin/org/jetbrains/research/testspark/services/TestStorageProcessingService.kt

```diff
@@ -375,4 +375,12 @@ class TestStorageProcessingService(private val project: Project) {
 
         return TestCase(testId, testName, testCode, setOf(), setOf(), setOf())
     }
+
+    private fun getJUnitVersion(): JUnitVersion {
```

Comment:
Same function in JUnitCombobox.kt.

Reply:
I can create an additional class for it, but I will work on a new version of `TestAssembler`, and I think, it's better to fix it there, because it will be much easier this a new assebler structure (I put a todo for myself)